installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -0,0 +1,131 @@
+## Preparing the Kubernetes namespace
+The deploy directory referenced below can be found in [the manageiq-pods repo](https://github.com/ManageIQ/manageiq-pods/tree/master/manageiq-operator/deploy)
+
+1. Search for the Custom Resource Definition (CRD) and create it if it doesn't already exist.
+```bash
+oc get crds | grep manageiqs.manageiq.org
+```
+```bash
+oc create -f deploy/crds/manageiq.org_manageiqs_crd.yaml
+```
+
+2. Set up RBAC.
+```bash
+oc create -f deploy/role.yaml
+oc create -f deploy/role_binding.yaml
+oc create -f deploy/service_account.yaml
+```
+
+3. Deploy the operator in your namespace.
+```bash
+$ oc create -f deploy/operator.yaml
+```
+
+## Migrating from Appliances
+
+### Collect data from the appliance
+1. Take a backup of the database
+```bash
+pg_dump -Fc -d vmdb_production > /root/pg_dump
+```
+
+2. Export the encryption key and Base64 encode it for Openshift
```

Comment:
OpenShift should have a capital S.  Additionally, we probably should be less OpenShift oriented where possible, so can this say Kubernetes instead (or Kubernetes/OpenShift)